README.md

```diff
@@ -8,6 +8,43 @@ A Google Maps JavaScript API v3 library to create and manage per-zoom-level clus
 
 Migrated from the [Google Maps JavaScript API utility libraries on Google Code](https://code.google.com/p/google-maps-utility-library-v3/).
 
+## Usage
+
+Download or clone `markerclusterer.js` and images `m1.png` to `m5.png`, save images in `images` folder.
+
+index.html
+
+    ...
+
+    <div id="map-container"><div id="map"></div></div>
+    <script src="markerclusterer.js"></script>
+    <script>
+        function initialize() {
+            var center = new google.maps.LatLng(51.5074, 0.1278);
+
+            var map = new google.maps.Map(document.getElementById('map'), {
+              zoom: 3,
+              center: center,
+              mapTypeId: google.maps.MapTypeId.ROADMAP
+            });
+
+            var markers = [];
+            var marker = new google.maps.Marker({
+                position: new google.maps.LatLng(51.5074, 0.1278)
+            });
+            markers.push(marker);
+
+            var options = {
```

Comment:
This is the default now, but I think it's still useful to demonstrate how a user could override with their own images. Could you add a comment to that effect?